docs/network-flow-visibility.md

```diff
@@ -171,6 +189,118 @@ through [Antrea Agent apiserver endpoint](prometheus-integration.md):
 `antrea_agent_conntrack_antrea_connection_count` and
 `antrea_agent_conntrack_max_connection_count`
 
+## Flow Aggregator
+
+Flow Aggregator is deployed as a Kubernetes Service. The main functionality of Flow
+Aggregator is to store, correlate and aggregate the flow records received from the
+Flow Exporter of Antrea Agents. More details on the functionality are provided in
+the [Supported Capabilities](#supported-capabilities-1).
+
+Flow Aggregator is implemented as IPFIX mediator, which
+consists of IPFIX Collector Process, IPFIX Intermediate Process and IPFIX Exporter
+Process. We use the [go-ipfix](https://github.com/vmware/go-ipfix) library to implement
+the Flow Aggregator.
+
+### Deployment
+
+For deployment of Flow Aggregator Service, we provide the manifest as part of the
+Antrea release bundle, and the manifest will also be available in the Antrea repository.
+If you are deploying the Flow Aggregator Service on a [vagrant setup](../test/e2e/README.md),
+you can use following command that deploys Antrea and Flow Aggregator in one go
+with the required configuration:
+
+```shell
+./infra/vagrant/push_antrea.sh -fc <externalFlowCollectorAddr>
+```
```

Comment:
this is really developer oriented

I don't think it makes sense to include this information, but not user-facing instructions on how to deploy the flow aggregator in their clusters (either a released version of the latest version from ToT)

Reply:
I added the manifest deployment steps here.

As there are a few steps involved in trying this feature, I thought pointing to a single script could be appealing for users, even though it is a vagrant setup. Moved these to a different section, which I feel more appropriate.